src/iotjs_binding.cpp

```diff
@@ -116,13 +119,17 @@ JObject::~JObject() {
 
 
 JObject& JObject::Null() {
-  static JObject null(JVal::Null(), false);
-  return null;
+  if (null == NULL) {
+    null = new JObject(JVal::Null(), false);
```

Comment:
I'm a bit concerned about this approach. If I'm right, "undefined reference to `__cxa_guard_{acquire,release}'" is a compiler/linker/build system problem in the first place. These are C++ ABI functions to guarantee thread-safe initialization of local statics. If their implementation is missing then something is wrong there.

I see two cases:

1) If thread safety is not an issue, i.e., `JObject::Null` cannot be invoked from multiple threads in parallel, then we can use the "-fno-threadsafe-statics" compiler option and then the guard calls will not be generated. (This is still a not-that-nice workaround, but it should work.)

2) However, if multiple threads can invoke this method, then we have just sidestepped thread synchronization with the proposed change, and we may end up allocating multiple orphaned objects on the heap. And if we don't want that, then we will have to add our own locking around the if/new code path. (BTW, the here-allocated objects never get freed.)